### ISSUE TITLE
errorsEquivalent showing blank/missing problem explanation

### ISSUE THREAD

**Issue by mhweiner** [mhweiner/hoare#8]
```
should throw for invalid strings ✖
   ✖

────────────────────────────────


────────────────────────────────


✖ 2/3 tests passed
```

**Comment by mhweiner** [comment]
<img width="439" alt="Screen Shot 2024-06-01 at 4 25 30 PM" src="https://github.com/mhweiner/hoare/assets/5946580/29268a2f-6b24-49f7-a562-9d07fcc8e1fd">

**Comment by mhweiner** [comment]
Fixed in 18a8765
